.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "stream" && "$BUILDKITE_STEP_KEY" == publish* ]]; then
+   echo ":: Fetching Pull Requests ::"
+   git config remote.origin.fetch "+refs/pull/*/head:refs/remotes/origin/pr/*"
+   git fetch
```

Comment:
@alexsapran I think we don't need this file anymore, can you confirm?

Reply:
Without .git/config I'll get "ambiguous argument" error from git. :(

Reply:
@dliappis can we proceed and rather add this to our "tech debt" todo list?

Reply:
@jlind23 yes sure